### ISSUE TITLE
Use window.phet.chipper.strings to extract string keys from live sims

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#224]
In https://github.com/phetsims/chipper/issues/883 I was asked to verify that Rosetta works with the sims that are built using ES6 modules.  It doesn't, and some changes are needed.  The problem is that `TranslationUtils.extractStringKeys` searches through the HTML5 code for the pattern "string!", which indicates usages of the string plugin.  This pattern is no longer present in the HTML5 files that are built using the ES6 modules approach.  Instead, this method should get the strings from the global variable `window.phet.chipper.strings`, which should work for both the pre- and post-ES6 files.

**Comment by jbphet** [comment]
Changes made, tested (some), and deployed.

I tested the new way of extracting strings by temporarily leaving the old way in the code, then comparing the two, and they were equal.  I then tested the new way on ES6 sims by adding temporary code that used ES6-built sims, see https://github.com/phetsims/chipper/issues/883#issuecomment-589742117.  These tests all passed when running Rosetta on my local Windows machine.  I also tested the "Test" function and submitting strings (to the test branch of Babel).

I deployed these changes to phet-server-dev, did some sanity testing, then deployed them to phet-server.  I did some sanity testing on phet-server as well, and all looked good.

I think we're done here.  Closing.
